proposals/0000-a11y-cross-platform.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Cross-platform Accessibility Improvements
+author:
+- Marc Mulcahy
+date: 2018-10-31
+---
+
+# RFC0000: Cross-platform Accessibility Improvements
+
+## Summary
+
+Several accessibility-related properties are only available on one platform (either iOS or Android, but not both). This proposal attempts to make as many properties available on both Android and iOS as possible.
+
+Properties only available on iOS include:
+
+* accessibilityViewIsModal
+* accessibilityElementsHidden
+
+Properties only available on Android include:
+
+* accessibilityLiveRegion
```

Comment:
This is also available on web

Reply:
We could standardize on ARIA's notion of a live region, and then dumb down the implementation to the various platforms. Android's notion of a live region is pretty basic (polite or assertive). iOS has no notion of live regions as far as I know, so we'd have to use iOS primitives to implement a subset of live region functionality.

Reply:
`accessibilityLiveRegion` is essentially already identical to ARIA live region, there's just a different in what the `none` state is called. So you could extend support for the existing API to iOS and then we'll have the 3 main platforms covered.

Reply:
+1 on the accessibilityLiveRegion property being more universal. We just implemented this for Windows (see PR [here](https://github.com/microsoft/react-native-windows/pull/2591)).

Reply:
+1 for accessibilityLiveRegion on iOS